### ISSUE TITLE
oada.get TypeError: Cannot set property 'tree' of undefined

### ISSUE THREAD

**Issue by maverick-zhn** [OADA/oada-cache#4]
```
Error in oada.get TypeError: Cannot set property 'tree' of undefined
    at Object._callee12$ (index.js:573)
    at tryCatch (runtime.js:63)
    at Generator.invoke [as _invoke] (runtime.js:290)
    at Generator.prototype.<computed> [as next] (runtime.js:116)
    at step (asyncToGenerator.js:21)
    at asyncToGenerator.js:32
```

**Comment by maverick-zhn** [comment]
The error happens here: 
```
                console.log('SENDING THIS REEVVVVVVVVVVVVVVV', response.data._rev);
                if (tree) watch.payload.tree = subTree;
```
I added this before the if(tree):
```javascript
watch.payload = {};
```
Apparantley that did it.

**Comment by tomohiroarakawa** [comment]
Which version of oada-cache are you using?

**Comment by maverick-zhn** [comment]
It is the `v2.1.0`

**Comment by tomohiroarakawa** [comment]
Yes, I agree.

**Comment by maverick-zhn** [comment]
I updated the `@oada/cerebral-provider`. It includes the `@oada/oada-cache` v3.0.0.
I am getting the same issue:
```
Error in oada.get TypeError: Cannot set property 'tree' of undefined
    at Object._callee12$ (index.js:589)
    at tryCatch (runtime.js:63)
    at Generator.invoke [as _invoke] (runtime.js:290)
    at Generator.prototype.<computed> [as next] (runtime.js:116)
    at step (asyncToGenerator.js:21)
    at asyncToGenerator.js:32
```
I am testing a get that includes watches to resources.
I presume the error is here:
```javascript
                info("Setting watch. Sending rev:", response.data._rev);
                if (tree) {
                  watch.payload.tree = subTree;
                }
```
I will debug the app. I send all the required arguments. At this point in the code, `watch.payload` is not defined.

Thanks.

**Comment by tomohiroarakawa** [comment]
I have published oada-cache v3.0.1. The issue should be fixed now. Let me know if the problem persists.
